data-prepper-core/src/integrationTest/resources/pipeline.yml

```diff
@@ -1,6 +1,10 @@
 entry-pipeline:
   source:
     otel_trace_source:
+  processor:
+    - peer_forwarder:
+        discovery_mode: "static"
+        data_prepper_ips: ["data-prepper1", "data-prepper2"]
```

Comment:
I'm making a minor change by renaming this property to "static_endpoints" in #222. The intent is to make it more clear that either hostnames _or_ IP addresses are valid for the static discovery mode. Sorry for not letting you know sooner.

Reply:
Now one of you should deal with merge conflict :P